scripts/regression/windows.ps1

```
@@ -399,16 +418,19 @@ function Check-JsonPure {
             break
         }
     }
+    if (-not (Test-AllowedExitCode -ExitCode $rc -AllowedExitCodes $AllowedExitCodes)) {
+        Write-Record $Name FAIL "$Note; exit=$rc"
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Write-Record' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)